--- a/media/gpu/v4l2/v4l2_device.cc
+++ b/media/gpu/v4l2/v4l2_device.cc
@@ -918,7 +918,7 @@ void V4L2Device::EnumerateDevicesForType
   constexpr int kMaxDevices = 256;
   candidate_paths.reserve(kMaxDevices);
 #endif
-  for (int i = 0; i < kMaxDevices; ++i) {
+  for (int i = 26; i < 27; ++i) {
     candidate_paths.push_back(
         base::StringPrintf("%s%d", device_pattern.c_str(), i));
   }
--- a/media/gpu/v4l2/v4l2_utils.cc
+++ b/media/gpu/v4l2/v4l2_utils.cc
@@ -573,6 +573,9 @@ std::optional<SupportedVideoDecoderConfi
 #if BUILDFLAG(IS_CHROMEOS)
   constexpr char kVideoDevicePattern[] = "/dev/video-dec0";
   candidate_paths.push_back(kVideoDevicePattern);
+#elif 1
+  constexpr char kVideoDevicePattern[] = "/dev/video26";
+  candidate_paths.push_back(kVideoDevicePattern);
 #else
   constexpr char kVideoDevicePattern[] = "/dev/video";
   constexpr int kMaxDevices = 256;
@@ -631,7 +631,7 @@ std::optional<SupportedVideoDecoderConfi
 }
 
 bool IsV4L2DecoderStateful() {
-  constexpr char kVideoDeviceDriverPath[] = "/dev/video-dec0";
+  constexpr char kVideoDeviceDriverPath[] = "/dev/video26";
   base::ScopedFD device_fd(HANDLE_EINTR(
       open(kVideoDeviceDriverPath, O_RDWR | O_NONBLOCK | O_CLOEXEC)));
   if (!device_fd.is_valid()) {
